tk_toolchain/cmd_line_tools/tk_build_qt_resources/__init__.py

```diff
@@ -169,30 +241,18 @@ def main():
             return 1
         print(f"Using PySide compiler version: {version}")
 
-    build_params = {
-        "qt_ui_path": args.qtuipath,
-        "py_built_path": args.pybuiltpath,
-        "import_text": args.importtext,
-    }
-
-    print("Building user interfaces...")
-    for index, ui_file in enumerate(args.uifiles):
-        build_params_ui_files = {
-            **build_params,
-            "compiler": args.uic,
-            "filename": ui_file,
-            "py_filename": args.uifilenames[index] if args.uifilenames else None,
-        }
-        build_qt_params = build_ui(**build_params_ui_files)
-        build_qt(**build_qt_params)
+    if args.yamlfile:
+        run_yaml_commands(args.yamlfile, args.uic, args.rcc)
+        return 1
 
-    print("Building resources...")
-    for index, res_file in enumerate(args.resfiles):
-        build_params_res_files = {
-            **build_params,
-            "compiler": args.rcc,
-            "filename": res_file,
-            "py_filename": args.resfilenames[index] if args.resfilenames else None,
-        }
-        build_qt_params = build_res(**build_params_res_files)
-        build_qt(**build_qt_params)
+    build_resources(
```

Comment:
I would get rid of this old/previous way and keep only one way: the yaml file way